Gemfile

```diff
@@ -1,11 +1,14 @@
 source 'https://rubygems.org'
 gem 'jekyll' # latest version
-gem 'uswds-jekyll', :git => 'https://github.com/18F/uswds-jekyll.git', :branch => 'main'
+gem 'mini_racer'
+gem 'bundler'
+gem 'rake'
```

Comment:
Do we need these included in the Gemfile?

Reply:
ah, we did for a prior version, then i migrated it to all node. I'll remove that

Reply:
mini_racer seems to have been added specifically to fix a JS bug:
https://github.com/18F/uswds-jekyll/commit/a6c2b5e3821ac4bd3851bb3fda4455f814ac5e04

Reply:
Let's remove all of them